sigs.yaml

```diff
@@ -3583,6 +3583,42 @@ workinggroups:
     liaison:
       github: aojea
       name: Antonio Ojea
+- dir: wg-checkpoint-restore
+  name: Checkpoint Restore
+  mission_statement: >
+    This working group aims to provide a central location for the community to discuss
+    the integration of Checkpoint/Restore functionality into Kubernetes.
+
+  charter_link: charter.md
+  stakeholder_sigs:
```

Comment:
This is a valuable initiative. The charter mentions that the scope includes checkpointing and restoring 'workloads' and providing 'guidance for developers on checkpoint-friendly app design.' Given this focus, it's essential for SIG Apps to be involved as a key stakeholder.

Reply:
Yes, thanks @janetkuo. I added SIG Apps to the proposal.